### ISSUE TITLE
Do you plan to release 320x512 resolution, 16 frames model?

### ISSUE THREAD

**Issue by ebartrum** [Drexubery/ViewCrafter#5]
Hi, thank you for the amazing work!
I was wondering if you plan to release your 320x512 resolution/16 frames model variant?
I have tried to use your 1024 resolution, 16 frame model to generate videos at 320 x 512, (using arguments --height 320 --width 512, and resizing the render_results) but observed texture artifacts!

**Comment by Drexubery** [comment]
Thanks! 
I will upload the 320x512 resolution model and provide the configs in the next couple of days; yet, I recommend you use the high-resolution model, as the 320x512 resolution model may exhibit texture flickering when handling high-frequency texture.

**Comment by ebartrum** [comment]
Thank you! I was also wondering, was your ViewCrafter model finetuned from the interpolation variant of DynamicCrafter, or the non-interpolation variant? If you have a 1024-resolution interpolation variant of DynamicCrafter, it would also be really useful if you can release that, as the currently released public DynamicCrafter interpolation model is limited to resolution 320x512!

**Comment by Drexubery** [comment]
> Thank you! I was also wondering, was your ViewCrafter model finetuned from the interpolation variant of DynamicCrafter, or the non-interpolation variant? If you have a 1024-resolution interpolation variant of DynamicCrafter, it would also be really useful if you can release that, as the currently released public DynamicCrafter interpolation model is limited to resolution 320x512!

Hi, ViewCrafter is finetuned from the non-interpolation variant. Unfortunately, we do not have a 1024-resolution interpolation model;

**Comment by ebartrum** [comment]
OK, thank you for letting me know!

**Comment by sudonymously** [comment]
@Drexubery quick question, do you know why the 320x512 model produces dramatically more artifacts? this is a problem with needing more training or is it something fundamental to the viewcrafter architecture?

**Comment by Drexubery** [comment]
> @Drexubery quick question, do you know why the 320x512 model produces dramatically more artifacts? this is a problem with needing more training or is it something fundamental to the viewcrafter architecture?

Hi, you can refer to the guide here https://github.com/Drexubery/ViewCrafter/issues/23#issuecomment-2371206533 to adjust the script to the 512 model, and see if it still produce artifacts.

**Comment by sudonymously** [comment]
https://github.com/user-attachments/assets/58f67fe4-f891-4e6b-bee1-144372cee7d3

I followed those steps and getting these results. is this expected?

**Comment by Drexubery** [comment]
> 
> 
> https://github.com/user-attachments/assets/58f67fe4-f891-4e6b-bee1-144372cee7d3
> 
> 
> 
> I followed those steps and getting these results. is this expected?

Yes, it generally looks fine, but as I mentioned, the 512 model suffers from significantly more severe texture flickering. Therefore, it is recommended to use the 1024 model.
